app/javascript/components/applications/application_form/AlternateContactSection.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react'
+import _ from 'lodash'
```

Comment:
bases on grant suggestions we are importing from lodash whatever we are actually using to help performance in our bundle